dev-cpp/cpp-ipc/files/cpp-ipc-1.3.0_p20250411-system-gtest.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7e07dd3..4e30a8b 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -51,7 +51,7 @@ if (LIBIPC_BUILD_TESTS)
+     else()
+         set(gtest_force_shared_crt ON)
+     endif()
+-    add_subdirectory(3rdparty/gtest)
++    find_package(GTest REQUIRED)
```

Comment:
Might be simpler to replace `3rdparty/gtest/CMakeLists.txt` via something like:

    echo "find_package(GTest REQUIRED)" > 3rdparty/gtest/CMakeLists.txt || die

…unless you're planning to send a fix upstream (adding a switch for it).